render.yaml

```
@@ -21,8 +21,9 @@ services:
           name: toolpad-db
           property: connectionString
       - fromGroup: toolpad-settings
-    ignoredPaths:
-     - docs/**
+    buildFilter:
+      ignoredPaths:
+        - docs/**
```

While we are at it?

```suggestion
        - docs/**
        - **/*.spec.ts
```

Curious we are not calling them "test.ts" 😁

---

> Curious we are not calling them "x.test.ts"

Muscle memory I think. I'm happy to rename, I don't have a strong opinion. I believe I saw both flavors in `@mui/material`

---

>  I believe I saw both flavors in @mui/material

True, the spec.tsx files are TypeScript tests.